lib/widgets/path_editor/cards/marker_card.dart

```diff
@@ -1,12 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:pathplanner/robot_path/robot_path.dart';
+import 'package:pathplanner/services/generator/trajectory.dart';
 import 'package:pathplanner/widgets/custom_input_slider.dart';
 import 'package:pathplanner/widgets/draggable_card.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 class MarkerCard extends StatefulWidget {
   final GlobalKey stackKey;
   final SharedPreferences prefs;
+  final RobotPath? path;
```

Comment:
This doesn't need to be nullable since the path in marker_editor cannot be null.

```suggestion
  final RobotPath path;
```